mautic/mautic
# Fix 191 characters for created lead fields

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for features or enhancements / 3.2 for bug fixes 
| Bug fix?                               | yes/no
| New feature?                           | yes/no
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 




#### Description:
In Mautic 3 we change length for indexed varchar columns to 191 https://github.com/mautic/mautic/pull/8349 
But If we create fields,  the column was created with 255 length.



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create new contact text field 
3. Check database if column in leads table has 191 length




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/9460"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
